### PR TITLE
implemented the homework in main.js

### PR DESCRIPTION
**Pull request by Heema5657**
bla bla bla done

### CODE REVIEW

File: javascript/javascript2/week2/Test.js [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+  {
+    title: "*batteries not included",
+    year: 1987,
+    rating: 6.6,
+    votes: 25636,
+    running_times: 6360,
+  },
+];
+// 1 - the movies with a short title
+let arrayOfShortMoviesTitle = movies
```
Comment (piyushpalod) [inline javascript/javascript2/week2/Test.js]: Just for the optimisation instant of calling a filter function 3 times call it once make the condition inside the filter. No need to call separate filter for other conditions.

File: javascript/javascript2/week2/Test.js [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+  }
+}).length;
+console.log(totalMoviesOfWords);
+//Create an array of movies where a word in the title is duplicated.
+
+let wordsSeparate = movies.map((movie) => {
+  const words = movie.title.toLowerCase().replace(/["']/g, "").split(" ");
+  let objectOfDuplicate = {};
+  let duplicateFound = false;
+  words.forEach((word) => {
```
Comment (piyushpalod) [inline javascript/javascript2/week2/Test.js:97]: Nice work 👍

**Review by piyushpalod — APPROVED** [review]

File: javascript/javascript2/week2/Test.js (hunk 1 above)
Comment (Heema5657) [reply javascript/javascript2/week2/Test.js]: Implemented the feedback changes. Kindly check and approve!

File: javascript/javascript2/week2/Test.js (hunk 2 above)
Comment (Heema5657) [reply javascript/javascript2/week2/Test.js:97]: Thanks !

**Pull request merged**